master/buildbot/worker/base.py

```diff
@@ -296,7 +297,10 @@ def stopService(self):
 
     def startMissingTimer(self):
         if self.notify_on_missing and self.missing_timeout and self.parent:
-            self.stopMissingTimer()  # in case it's already running
+            try:
+                self.stopMissingTimer()  # in case it's already running
+            except error.AlreadyCalled:
```

Comment:
Yeah, I thought about that as well, but that would hide if stopMissingTimer fails unexpected. Currently failing in worker start should be expected since we have no way of querying in which state the `callLater` instance is. 

I'm running on top of yesterdays master, so should be new enough?